ui/src/components/badge/__tests__/QBadge.spec.js

```diff
@@ -1,73 +1,162 @@
-// import { mount } from '@cypress/vue'
-// import QBadge from '../QBadge.js'
+import { mount } from '@cypress/vue'
+import QBadge from '../QBadge.js'
 
-// const snapshotOptions = { customSnapshotsDir: '../src/components/badge/__tests__' }
+const defaultOptions = {
+  label: 'simple badge'
+}
+
+function mountQBadge (options = {}) {
+  options.props = {
+    ...defaultOptions,
+    ...options.props
+  }
+
+  return mount(QBadge, options)
+}
 
 describe('Badge API', () => {
   describe('Props', () => {
     describe('Category: content', () => {
       describe('(prop): floating', () => {
-        it.skip(' ', () => {
-          //
+        it('should render a floating badge', () => {
+          mountQBadge({
+            props: { floating: true }
+          })
+
+          cy.get('.q-badge')
+            .should('have.class', 'q-badge--floating')
         })
       })
 
       describe('(prop): multi-line', () => {
-        it.skip(' ', () => {
-          //
+        it('should render a content with multiple lines', () => {
+          mountQBadge({
+            props: { multiLine: true }
+          })
+
+          cy.get('.q-badge')
+            .should('have.class', 'q-badge--multi-line')
         })
       })
 
       describe('(prop): label', () => {
-        it.skip(' ', () => {
-          //
+        it('should render a label inside the badge', () => {
+          const label = 'Badge label'
+
+          mountQBadge({
+            props: { label }
+          })
+
+          cy.get('.q-badge')
+            .should('contain', label)
         })
       })
 
       describe('(prop): align', () => {
-        it.skip(' ', () => {
-          //
+        it('should render a badge aligned to the top when align is set to "top"', () => {
```

Comment:
Here you can use a for loop to test the prop for each acceptable value, since the test is pretty simple and all options have similar effects